CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+v0.8.0
+------
+* **Release date**: 2016-03-04.
+* Fixed migration loop to handle the case where there are no explicit migrations to run.
+* Railway route extraction no longer relies on the `planet_osm_nodes` table, which may be missing if flat nodes is enabled.
```

Comment:
This has the benefit of including more data in the calculations, promoting the right stations to a better `tile_kind_rank`.